corehq/apps/userreports/tests/__init__.py

```diff
@@ -1,36 +0,0 @@
-from .test_app_manager_integration import *
```

Comment:
is this you proposing we should remove all the imports from tests `__init__` files because of the switch to nose? was this causing problems or did you just decide to throw in this PR?